Author: José Manuel Santamaría Lema <panfaust@gmail.com>,
        Maximiliano Curia <maxy@debian.org>
Description: This patch adds a nlist.h wrapper include for [k]FreeBSD systems.
 This include was making kde-workspace FTBFS on gnu/kfreebsd including kvm.h
 or nlist.h directly, this wrapper includes bsd/nlist.h. The patch should be
 removed once libbsd-dev and libkvm-dev fix their interaction.
Index: kde-workspace-4.11.3/ksysguard/ksysguardd/FreeBSD/CMakeLists.txt
===================================================================
--- kde-workspace-4.11.3.orig/ksysguard/ksysguardd/FreeBSD/CMakeLists.txt	2013-06-28 19:10:44.349417000 +0200
+++ kde-workspace-4.11.3/ksysguard/ksysguardd/FreeBSD/CMakeLists.txt	2013-11-09 02:39:05.000000000 +0100
@@ -1,4 +1,5 @@
 INCLUDE_DIRECTORIES( ${CMAKE_CURRENT_SOURCE_DIR}/../CContLib ${CMAKE_CURRENT_SOURCE_DIR}/../ )
+include_directories( ${CMAKE_CURRENT_SOURCE_DIR} )
 
 ADD_LIBRARY(libksysguardd STATIC
             acpi.c
@@ -12,3 +13,4 @@
             ProcessList.c
             stat.c
             uptime.c)
+
Index: kde-workspace-4.11.3/ksysguard/ksysguardd/FreeBSD/nlist.h
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ kde-workspace-4.11.3/ksysguard/ksysguardd/FreeBSD/nlist.h	2013-11-09 04:41:55.000000000 +0100
@@ -0,0 +1,4 @@
+#ifndef KDE_W_NLIST_H_HACK
+#define KDE_W_NLIST_H_HACK
+#include <bsd/nlist.h>
+#endif
